publications.md

```diff
@@ -188,7 +193,8 @@ If you have used or extended KLEE and would like to have your paper listed here,
    IEEE Trustcom/BigDataSE/ISPA 2015
    August 20-22, 2015, Helsinki, Finland
 
-1. [Angelix: Scalable Multiline Program Patch Synthesis via Symbolic Analysis](http://www.comp.nus.edu.sg/~abhik/pdf/ICSE16-angelix.pdf)
```

Comment:
@mechtaev Please rebase this PR on the master, since this seems to be a spurious change.